netlify.toml

```diff
@@ -1,12 +1,12 @@
 # example netlify.toml
 [build]
-  command = "vite build"
+  command = "npm run build"
   functions = "netlify/functions"
   publish = "dist"
 
   ## Uncomment to use this redirect for Single Page Applications like create-react-app.
   ## Not needed for static site generators.
-  #[[redirects]]
+  [[redirects]]
   #  from = "/*"
```

Comment:
This section shouldn't be commented as it enables reloading pages in single page applications. Currently, if we reload any page, we'll get not found.